app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -288,20 +267,21 @@ private fun ProfileImage(
 }
 
 @Composable
-private fun FavoriteParkingsSection(
-    favoriteParkings: List<Parking>,
-    onFavoritesUpdated: (List<Parking>) -> Unit
-) {
+private fun FavoriteParkingsSection(userViewModel: UserViewModel) {
+  val favoriteParkings = userViewModel.favoriteParkings.collectAsState().value
```

Comment:
This line could be simplified by: 
`val favoriteParkings by userviewModel.favoriteParkings.collectAsState()`
Although it might require an import

Reply:
no it cant ?

Reply:
or do you expect me to call favoriteParkings.value everywhere ? I dont know why i would do that though...

Reply:
The `by` key-word "extracts" the value directly, it's the delegation concept of Kotlin. The value of the variable is delegated to the state.
But it's a minor detail nevermind😅